src/compiler/diagnosticMessages.json

```diff
@@ -647,6 +647,10 @@
         "category": "Error",
         "code": 1208
     },
+    "Invalid optional chain from new expression.": {

```

Comment:
Could we have a suggestion here as well? At least in the case from the example, we could probably suggest something like `Did you forget to call 'A'?`

Reply:
@gabritto I think we can. `Invalid optional chain from new expression. Did you mean to call {0}()?` Do I need to add it?

Reply:
Yes, I think it would be nice to add it.

Reply:
Oke.